Entities/ACrab.h

```diff
@@ -585,6 +597,7 @@ int FirearmActivationDelay() const;
 	float m_JetTimeTotal;
 	// How much time left the jetpack can go, in ms
 	float m_JetTimeLeft;
+	float m_JetReplenishRate; //!< How fast the jetpack fuel will replenish when not in use, in relation to the legacy value.
```

Comment:
This comment should change slightly, since the rate is now a multiplier where 1.0 is normal, 2.0 is double, etc.
I guess the whole thing could be renamed to `JetReplenishRateMultiplier` or something if you wanted, but I'm not picky there.

Reply:
Should it also change for INI, is `JumpReplenishRateMultiplier` too much of a mouthful?

Reply:
Eugh, I dunno tbh. We're back in naming nightmares I guess lol
I lean towards long descriptive names, but I know that people sometimes dislike that (especially in ini) cause it means more typing or whatever. We could do `JumpReplenishMultiplier` but that sounds weird to me.

Reply:
I'll leave it as-is, we can deal with this later if we must

Reply:
Works for me :)